plugin-dev/Source/Sentry/Private/Apple/SentrySubsystemApple.cpp

```diff
@@ -303,3 +302,36 @@ USentryTransaction* SentrySubsystemApple::StartTransactionWithContextAndOptions(
 
 	return SentryConvertorsApple::SentryTransactionToUnreal(transaction);
 }
+
+USentryTransactionContext* SentrySubsystemApple::ContinueTrace(const FString& sentryTrace, const TArray<FString>& baggageHeaders)
```

Comment:
Can we add some tests for this?

Reply:
Yeah, I've added a basic one. The existing `sentry-native` API is quite limited though so it's somewhat problematic to set up thorough testing for the tracing feature.